bindings/wasm/src/did/wasm_document.rs

```diff
@@ -153,6 +313,39 @@ impl WasmDocument {
     method.revoke_merkle_key(index).wasm_result()
   }
 
+  /// Attaches the relationship to the given method, if the method exists.
+  ///
+  /// Note: The method needs to be in the set of verification methods,
+  /// so it cannot be an embedded one.
+  #[wasm_bindgen(js_name = attachMethodRelationships)]
+  pub fn attach_method_relationships(
+    &mut self,
+    did_url: &WasmDIDUrl,
+    relationship: &MethodRelationshipType,
+  ) -> Result<()> {
+    let relationship: WasmMethodRelationship = relationship.into_serde().wasm_result()?;
+    self
+      .0
+      .attach_method_relationship(&did_url.0, relationship.into())
+      .wasm_result()?;
+    Ok(())
+  }
+
+  /// Detaches the given relationship from the given method, if the method exists.
+  #[wasm_bindgen(js_name = detachMethodRelationships)]
+  pub fn detach_method_relationships(
```

Comment:
This should be singular after the latest changes.

```suggestion
  #[wasm_bindgen(js_name = detachMethodRelationship)]
  pub fn detach_method_relationship(
```